src/main/java/seedu/duke/WeekView.java

```diff
@@ -0,0 +1,33 @@
+package seedu.duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class WeekView {
+    private LocalDate startOfWeek;
+    private final DateTimeFormatter dateFormatter;
```

Comment:
Good usage of final, preventing any unnecessary accidental changes in the future!